src/control.rs

```diff
@@ -24,6 +26,8 @@ impl Scheduler {
         Self {
```

Comment:
The `Scheduler` is the driver of the `AudioScheduledSourceNode` trait which corresponds to https://www.w3.org/TR/webaudio/#AudioScheduledSourceNode
We should probably move offset & duration to the `Controller`. The Controller (not part of the spec) is also used for MediaElements but I will figure out a way to actually use these fields there later